__tests__/article.test.js

```diff
@@ -0,0 +1,64 @@
+const request = require('supertest')
+const app = require('../app')
+const testData = require('../db/data/test-data')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe('GET /api/articles/:article_id', () => {
+    it('should respond with the correct article object when given a valid article ID', () => {
+      return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({ body }) => {
+                expect(body.article).toHaveProperty('article_id', expect.any(Number))
```

Comment:
We can be more specific with our expect here, as we know we're expecting the number 1